lib/brainstem/api_docs/introspectors/rails_introspector.rb

```diff
@@ -175,6 +176,24 @@ def base_controller_class
         attr_writer :base_controller_class
 
 
+        #
+        # Returns a proc that will return the application or engine to get routes from.
+        #
+        # @return [Proc] Proc that returns the Rails application or an engine
+        #
+        def base_application_proc
+          @base_application_proc ||= Proc.new { Rails.application }
+        end
```

Comment:
Duplicating the default value is a little suspect... but is also how the other methods in here do it.